components/ws-daemon/pkg/content/service.go

```diff
@@ -153,10 +153,7 @@ func (s *WorkspaceService) InitWorkspace(ctx context.Context, req *api.InitWorks
 	var (
 		wsloc string
 	)
-	if req.FullWorkspaceBackup {
-		if s.runtime == nil {
-			return nil, status.Errorf(codes.FailedPrecondition, "full workspace backup is not available - not connected to container runtime")
-		}
+	if req.FullWorkspaceBackup || req.PersistentVolumeClaim {
 		var mf csapi.WorkspaceContentManifest
```

Comment:
PVC workspaces have no content manifest - I'd be surprised if this code worked.

Reply:
Indeed it doesn't. I will remove it in a follow up PR (which is currently in draft state)